arcgis-ios-sdk-samples/Route and directions/Display route layer/DisplayRouteLayerViewController.swift

```diff
@@ -0,0 +1,86 @@
+            mapView.map = makeMap()
+        }
+    }
+    
+    @IBOutlet var directionsBarButtonItem: UIBarButtonItem!
+    
+    var directions = [String]()
+    
+    func makeMap() -> AGSMap {
+        // Set the basemap.
+        let map = AGSMap(basemapStyle: .arcGISTopographic)
+        // Set the portal.
+        let portal = AGSPortal.arcGISOnline(withLoginRequired: false)
+        // Create the portal item with the item ID for route data in Portland, OR.
+        let item = AGSPortalItem(portal: portal, itemID: "0e3c8e86b4544274b45ecb61c9f41336")
+        // Create a collection of features using the item.
+        let featureCollection = AGSFeatureCollection(item: item)
+        // Load the feature collection.
+        loadFeatureCollection(featureCollection)
+        // Create a feature collection layer using the feature collection.
+        let featureCollectionLayer = AGSFeatureCollectionLayer(featureCollection: featureCollection)
+        // Set the feature collection layers to the map's operational layers.
+        map.operationalLayers.setArray([featureCollectionLayer])
+        // Set the viewpoint.
+        let viewpoint = AGSViewpoint(latitude: 45.2281, longitude: -122.8309, scale: 57e4)
+        map.initialViewpoint = viewpoint
+        return map
+    }
+    
+    func loadFeatureCollection(_ featureCollection: AGSFeatureCollection) {
+        // Load the feature collection.
+        featureCollection.load { [weak self] error in
+            guard let self = self else { return }
+            // Present an error if loading was unsuccessful.
+            if let error = error {
+                self.presentAlert(error: error)
+            } else {
+                // Make an array of all the feature collection tables.
+                let tables = featureCollection.tables as! [AGSFeatureCollectionTable]
+                // Get the table that contains the turn by turn directions.
+                let directionsTable = tables.first(where: { $0.tableName == "DirectionPoints" })
+                // Create an array of all the features in the table.
+                let features = directionsTable?.featureEnumerator().allObjects
+                // Set the array of directions.
+                self.directions = features?.compactMap { $0.attributes["DisplayText"] } as! [String]
+                // Enable the directions bar button item.
+                self.directionsBarButtonItem.isEnabled = true
+            }
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Add the source code button item to the right of navigation bar.
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["DisplayRouteLayerViewController", "DisplayDirectionsViewController"]
+    }
+    
+    // MARK: - Navigation
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if let controller = segue.destination as? DisplayDirectionsViewController {
```

Comment:
Please refer to the contribution guideline `wiki/iOS-Samples-Repo:-A-Contribution-Guide#segue-actions` and make this a segue action. For example: https://github.com/Esri/arcgis-runtime-samples-ios/pull/1127/files#diff-9496ca21636473d62bcaa93a63c59628b0fb75b46f39ce0095a52133689073ffR74

<img width="314" alt="image" src="https://user-images.githubusercontent.com/9660181/167523383-64e50030-b81f-47ed-b001-84d4a7952f3c.png">